include/kernels/multi_reduce_core.cuh

```diff
@@ -15,98 +15,101 @@ namespace quda
     /**
        @brief Parameter struct for generic multi-blas kernel.
        @tparam NXZ is dimension of input vectors: X,Z,V
-       @tparam NXZ is dimension of input vectors: X,Z
        @tparam store_t Default store type for the fields
        @tparam N Default field vector i/o length
        @tparam y_store_t Store type for the y fields
        @tparam N Y-field vector i/o length
        @tparam Reducer Functor used to operate on data
     */
-    template <int NXZ_, typename store_t, int N, typename y_store_t, int Ny, typename Reducer_>
+    template <typename real_, int n_, int NXZ_, typename store_t, int N, typename y_store_t, int Ny, typename Reducer_>
```

Comment:
Update doxygen above to reflect new order of arguments, new argument `n_` and `real_`

Reply:
172f419b3